HoloPipelines/pipelines/components/lungSegment/segment_airway.py

```diff
@@ -0,0 +1,59 @@
+import numpy as np
+import nibabel as nib
+from scipy import ndimage
+import pathlib
+
+try:
+    from utils import close_space_dilation, generate_structure_trachea, generate_initLoc
+except Exception as e:
```

Comment:
local variable 'e' is assigned to but never used. Log the exception that was caught.

Reply:
'e' now used in logging